.github/workflows/deploy.yaml

```diff
@@ -17,15 +17,9 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
-        with:
```

Comment:
We don't need fetch depth 0. Just a waste of resources.

Reply:
Don't we need it to count the number of commits here?

Reply:
You might be right. This line would require fetch depth 0?
https://github.com/cursorless-dev/cursorless/blob/0795f9d52848bff38617e65dbd6bc47ad59bd540/packages/cursorless-vscode/src/scripts/populateDist/transformPackageJson.ts#L27

Reply:
Restored